sw/device/rom_exts/manifest.h

```diff
@@ -0,0 +1,220 @@
+// Copyright lowRISC contributors.
```

Comment:
Why are we checking in the file, rather than generating it at build time?

Reply:
That is a good point, I haven't thought about it.

Reply:
Garret and I discussed this, and we would prefer it to be checked in, like the top_earlgrey headers are.

Reply:
Done, however not couldn't get meason to do exactly what I wanted. Maybe you could suggest how I can tell the generator to rebuild the header if one of N files has changed (haven't pushed the updates yet).

Reply:
Hmmm okay, well I suppose I can remove the commit.

Reply:
So with the latest change it is still checked in, but also auto generated.